### ISSUE TITLE
Apparent Escaping Issue with Basepath Dirname/Python Call

### ISSUE THREAD

**Issue by majascules** [fritz-hh/OCRmyPDF#109]
Fritz,

I am not a programmer, and yet I find myself using your script. Please understand and excuse any breaches of protocol or style. Thank you for your work. I appreciate it.

I'm running bash on OSX (Yosimite). GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin14)

I called your script ./OCRmyPDF.sh and it gave me an error:
```
usage: dirname path
./OCRmyPDF.sh: line 12: /src/config.sh: No such file or directory
```
I then ran with bash -x and got the following hint:
```
./OCRmyPDF.sh: line 12: /src/config.sh: No such file or directory
albedo:OCRmyPDF-2.x esmaeilkhaksari$ bash -x ./OCRmyPDF.sh 
+ command -v python2
+++ python2 -c 'import os; print os.path.realpath("./OCRmyPDF.sh")'
++ dirname /Users/esmaeilkhaksari/Google Drive/code/pdf2pdf/OCRmyPDF-2.x/OCRmyPDF.sh
usage: dirname path
+ BASEPATH=
+ echo ''

+ . /src/config.sh
./OCRmyPDF.sh: line 12: /src/config.sh: No such file or directory
```
I suspected this might have something to do with the space in my filename. Not really knowing what to do about this, I tried to wrap the string in double quotes to see if this would fix the issue, and it has.

original:
```
BASEPATH="$(dirname $(python2 -c "import os; print os.path.realpath(\"$0\")"))"
```

amendment:
```
BASEPATH="$(dirname "$(python2 -c "import os; print os.path.realpath(\"$0\")")")"
````

I hope this is helpful.

Best regards,
Esmaeil Khaksari

**Comment by jbarlow83** [comment]
Try creating a symlink for python2.
```
sudo ln -s $(which python) /usr/local/bin/python2
```

**Comment by majascules** [comment]
$ sudo ln -s $(which python) /usr/local/bin/python2
ln: /usr/local/bin/python2: File exists

**Comment by jbarlow83** [comment]
Fixed in v3.0-rc2
